workflows/create-pull-request.yaml

```diff
@@ -0,0 +1,59 @@
+name: Create Pull request
+
+on:
+  schedule:
+    # Runs "at minute 44 past midnight" (see https://crontab.guru)
+    - cron: "44 0 * * *"
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    env:
+      bot_username: BotName
+      bot_email: BotMail@gmail.com
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ["de-german", "fr-french"]
+    steps:
+      - name: Fetch translation repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+          repository: solidity-docs-test/${{ matrix.repos }}
+          fetch-depth: "0"
+
+      - name: Fetch this repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+          repository: solidity-docs/translation-guide
+          path: .github-workflow
```

Comment:
Can you explain this?

Reply:
@chriseth 
Yes, indeed, this part is not really intuitive. I need to fetch the repository for external shell scripts. The flow will look like this:
- fetch translation repository
- fetch merge script (from the same repository as GitHub Action)
- fetch original repository and merge it to translation repository
- publish sync branch with a merge commit
- create a pull request

Reply:
@cameel 
I am using Bot Personal Access Token. I wanted the PR to be created by customized "Solidity PR Bot" and not generic "GitHub Action". Is this ok?

Reply:
As we agreed on the call, it should work both with and without a dedicated account for the bot. We'll probably use it without an account but we can keep an option open to start using one later.